ruff.toml

```diff
@@ -0,0 +1,2 @@
+[lint]
+ignore = ["E402","E703"]
```

Comment:
```suggestion
[lint.per-file-ignores]
**.ipynb = ["E402", "E703"]
```

Reply:
okay it work if we use   double quote and then **.ipynb inside it.
```
[lint.per-file-ignores]
"**.ipynb" = ["E402", "E703"]

```